bazel/grpc_deps.bzl

```diff
@@ -147,16 +147,15 @@ def grpc_deps():
     if "com_github_gflags_gflags" not in native.existing_rules():
         http_archive(
             name = "com_github_gflags_gflags",
-            strip_prefix = "gflags-30dbc81fb5ffdc98ea9b14b1918bfe4e8779b26e",
```

Comment:
the current git submodule for gflags is at `30dbc81fb5ffdc98ea9b14b1918bfe4e8779b26e` - is it not problematic to have the bazel dep and the submodule out of sync?

Reply:
I'm a bit concerned about the bazel and git submodule dependency versions diverging - from looking, all dependencies existing as both bazel deps and submodules are currently at the same version (with the exception of abseil cpp). This would create a new divergence

Reply:
dependency bumped per your request